cluster/charts/crossplane/values.yaml

```diff
@@ -1,213 +1,218 @@
+  # - A policy of `All` includes all the `Basic` settings and also creates Crossplane Roles in all namespaces.
   managementPolicy: All
-  # -- Enable leader election for RBAC Managers pod.
+  # -- Enable leader election for the RBAC Manager pod.
   leaderElection: true
-  # -- A list of additional args to be pased to the RBAC manager's container.
+  # -- Add custom arguments to the RBAC Manager pod.
   args: []
-  # -- Enable nodeSelector for RBAC Managers pod.
+  # -- Add `nodeSelectors` to the RBAC Manager pod deployment.
   nodeSelector: {}
-  # -- Enable tolerations for RBAC Managers pod.
+  # -- Add `tolerations` to the RBAC Manager pod deployment.
   tolerations: []
-  # -- Enable affinity for RBAC Managers pod.
+  # -- Add `affinities` to the RBAC Manager pod deployment.
   affinity: {}
 
-# -- Priority class name for Crossplane and RBAC Manager (if enabled) pods.
+# -- The PriorityClass name to apply to the Crossplane and RBAC Manager pods.
 priorityClassName: ""
 
 resourcesCrossplane:
   limits:
-    # -- CPU resource limits for Crossplane.
+    # -- CPU resource limits for the Crossplane pod.
     cpu: 100m
-    # -- Memory resource limits for Crossplane.
+    # -- Memory resource limits for the Crossplane pod.
     memory: 512Mi
   requests:
-    # -- CPU resource requests for Crossplane.
+    # -- CPU resource requests for the Crossplane pod.
     cpu: 100m
-    # -- Memory resource requests for Crossplane.
+    # -- Memory resource requests for the Crossplane pod.
     memory: 256Mi
 
 securityContextCrossplane:
-  # -- Run as user for Crossplane.
+  # -- The user ID used by the Crossplane pod.
   runAsUser: 65532
-  # -- Run as group for Crossplane.
+  # -- The group ID used by the Crossplane pod.
   runAsGroup: 65532
-  # -- Allow privilege escalation for Crossplane.
+  # -- Enable `allowPrivilegeEscalation` for the Crossplane pod.
   allowPrivilegeEscalation: false
-  # -- ReadOnly root filesystem for Crossplane.
+  # -- Set the Crossplane pod root file system as read-only.
   readOnlyRootFilesystem: true
 
 packageCache:
-  # -- Storage medium for package cache. `Memory` means volume will be backed by tmpfs, which can be useful for development.
+  # -- Set to `Memory` to hold the package cache in a RAM-backed file system. Useful for Crossplane development.
   medium: ""
-  # -- Size limit for package cache. If medium is `Memory` then maximum usage would be the minimum of this value the sum of all memory limits on containers in the Crossplane pod.
+  # -- The size limit for the package cache. If medium is `Memory` the `sizeLimit` can't exceed Node memory.
   sizeLimit: 20Mi
-  # -- Name of the PersistentVolumeClaim to be used as the package cache. Providing a value will cause the default emptyDir volume to not be mounted.
+  # -- The name of a PersistentVolumeClaim to use as the package cache. Disables the default package cache `emptyDir`.
```

Comment:
I found it a bit confusing, maybe:
```suggestion
  # -- The name of a PersistentVolumeClaim to use as the package cache. Setting it to `emptyDir` disables the default package cache.
```

Reply:
I totally misunderstood it, sorry, but yes, adding `Volume` would make it clearer 👍

Reply:
Fixed!